### PR TITLE
fix(config): correct Evolve LRM-AS

### PR DESCRIPTION
**Pull request by cdnninja**
LRM-AS is the same as WD500Z.  Updating to reflect same functionality.

### CODE REVIEW

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 1]
```diff
@@ -17,15 +17,23 @@
 	},
 	"associations": {
 		"1": {
-			"label": "Group 1",
+			"label": "Controller",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: This should be called Lifeline. Please check the style guide in the documentation and ensure that these changes conform.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 2]
```diff
@@ -36,18 +44,62 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No",
+					"label": "Use Start Level",
 					"value": 0
 				},
 				{
-					"label": "Yes",
+					"label": "Ignore Start Level",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: This should be reworded so the options can be enable/disable per the style guide.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 3]
```diff
@@ -36,18 +44,62 @@
...
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED ON - Switch ON",
+					"value": 0
+				},
+				{
+					"label": "LED ON - Switch OFF",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: The description doesn't describe what is happening in the options.

**Review by blhoward2 — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0113/lrm-as.json (hunk 3 above)
Comment (cdnninja) [reply packages/config/config/devices/0x0113/lrm-as.json]: I will update and re-submit. I copied this line for line from the existing WD500z handler so I may update that one if I get this one approved. I suspect it also doesn't meet standards.

File: packages/config/config/devices/0x0113/lrm-as.json (hunk 2 above)
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: This is still a needed fix. Reword the label so the options can be Enable/Disable per the style guide.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 4]
```diff
@@ -36,18 +44,62 @@
...
+				{
+					"label": "Disable",
 					"value": 1
 				}
 			]
 		},
 		"4": {
 			"label": "Invert Switch",
-			"description": "Change the top of the switch to OFF and the bottom of the switch to ON.",
+			"description": "Reverse switch direction",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: Delete unnecessary description.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 5]
```diff
@@ -36,18 +44,62 @@
...
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"14": {
+			"label": "Enable Shade Control Group 2",
+			"description": "The switch can control shade control devices if this parameter is set to 1",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: Remove unecessary description.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 6]
```diff
@@ -58,15 +110,63 @@
...
 				{
-					"label": "Yes",
+					"label": "Enable",
 					"value": 1
 				}
 			]
 		},
+		"15": {
+			"label": "Enable Shade Control Group 3",
+			"description": "The switch can control shade control devices if this parameter is set to 1.",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: Ditto

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 7]
```diff
@@ -36,18 +44,62 @@
...
+					"value": 1
+				}
+			]
+		},
+		"14": {
+			"label": "Enable Shade Control Group 2",
+			"description": "The switch can control shade control devices if this parameter is set to 1",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 255,
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: If the options are 0/1 the max should be 1 and not 255.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 8]
```diff
@@ -58,15 +110,63 @@
...
 					"value": 1
 				}
 			]
 		},
+		"15": {
+			"label": "Enable Shade Control Group 3",
+			"description": "The switch can control shade control devices if this parameter is set to 1.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: Same note about the max value.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 9]
```diff
@@ -58,15 +110,63 @@
...
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"19": {
+			"label": "LED Transmission Indication",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: Change to "LED Transmission Indicator"

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 10]
```diff
@@ -58,15 +110,63 @@
...
+			"description": "LED mode while transmitting",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: Remove unnecessary description.

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 11]
```diff
@@ -58,15 +110,63 @@
...
+					"value": 1
+				},
+				{
+					"label": "Flicker for 1 second",
+					"value": 2
+				}
+			]
+		},
 		"29": {
 			"label": "Load Sense",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json:165]: Change to "Load Sensing"

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 12]
```diff
@@ -58,15 +110,63 @@
...
 			"valueSize": 1,
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json:166]: Fix the max value as above.

**Review by blhoward2 — CHANGES_REQUESTED** [review]
You're still missing a bunch of things in the style guide. Please correct and request a new review when ready. https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide

**Comment by cdnninja** [comment]
Yes for sure.  

I plan to finish it up when I get time. Thanks for following up.

**Comment by cdnninja** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0113/lrm-as.json [hunk 13]
```diff
@@ -17,15 +17,23 @@
...
+			"maxNodes": 5
 		}
 	},
 	"paramInformation": {
-		"3": {
-			"label": "Night Light",
-			"description": "In night-light mode the LED will turn ON when the switch is turned OFF.",
+		"1": {
+			"label": "Dim Start Level",
+			"description": "Start dimming from that start level or current level",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0113/lrm-as.json]: This description doesn't make sense. Is it the start level or the current level? Which are you setting by chosing enable?

**Review by blhoward2 — CHANGES_REQUESTED** [review]

**Review by blhoward2 — APPROVED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**